src/jobs/release-package.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+description: >
+  Signs and publishes npm package
+  The following environment variables are required:
+  - NPM_TOKEN
+  - SALESFORCE_KEY
+  - AWS_ACCESS_KEY_ID
+  - AWS_DEFAULT_REGION
+  - AWS_SECRET_ACCESS_KEY
+
+parameters:
+  salesforce_key:
+    description: salesforce key
+    type: string
+  npm_token:
+    description: npm token
+    type: string
+  key_path:
+    description: path to salesforce key
+    default: /root/salesforce-cli.key
+    type: string
+
+executor: default
+
+steps:
+  - checkout
+  - restore_cache:
+      keys:
+        - v1-npm-{{checksum ".circleci/config.yml"}}-{{checksum "yarn.lock"}}
+        - v1-npm-{{checksum ".circleci/config.yml"}}
+  - run:
+      name: Install dependencies
+      command: yarn
```

Comment:
For example with the cache, right after this yarn command you would save the cache.